### ISSUE TITLE
Stop changing the order of my palettes

### ISSUE THREAD

**Issue by RichardJECooke** [musescore/MuseScore#16660]
### Issue type

UI bug

### Bug description

I keep dragging my palettes into alphbetical order and when I reopen Musescore it puts them back in its preferred order. Please stop this.

### Steps to reproduce

Put palettes in alphabetical order.
Reopen Musescore 4. Perhaps a few times. It might take a while to happen

### MuseScore Version

OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.0.1-230121751, revision: github-musescore-musescore-9b70a8c

### Regression

I don't know

### Operating system

Windows 10

**Comment by MarcSabatella** [comment]
FWIW, I have a custom order and have never seen MusesScore change this.  But a couple of things that come to mind are, perhaps you are using Help / Revert to factory settings for some reason, not realizing that this is one of the settings that will be reverted.  Or you might have changed worksapces, not realizing that palette configuration is a part of that.  Or you might have a permissions problem in your home folder preventing MuseScore from recording your preference.  Are other preferences - like ones in Edit / Preferences - remembered?

In general, it's better to use the official Support forum on musescore.org when asking for help with problems.  Then if something is confirmed to be a bug and is reproducible, only then would it make sense to open an issue here on GitHub.

**Comment by RichardJECooke** [comment]
- I have not factory reset
- I don't know what workspaces are and have never used one
- My keyboard shortcut change is remembered. But I cannot change canvas / wallpapers.
- I will try to reproduce it and log specific instructions

**Comment by M00nbox** [comment]
I can confirm this behavior with MuseScore 4.01 on a system running Windows 10 64-bit. Alphabetically re-order palettes, all good. Close MuseScore. Reopen MuseScore. Palette order has reverted to default, which made little sense to user in the first place. User, realizing effort of alphabetization was wasted, gets annoyed.

Like RichardJECooke, I had not messed with factory settings or workspace. Simply tried to re-order palettes in a way that made them more intuitively useful & findable. This action had worked fine in MuseScore 3.6.2 -- needed to be done only once, behaved as expected upon closing/reopening MuseScore.

I do appreciate the care & dedication of all who labor to upgrade the app.

**Comment by M00nbox** [comment]
By "window," Marc, are you referring to tabbed scores? Yes, I often have a few scores or transcribed tunings of scores open at the same time, but always within a single instance of MuseScore. And anyway, in v3.6.2 a change in palette order made in any open tabbed score seemed to propagate OK to all other scores (altho I can understand why this may not always be desired).

Certainly MuseScore has to start out with _some_ palette order, which any user is free to customize to suit their preferred working style. Me, I can't possibly remember a lengthy non-alphabetic list of categories, whether or not it makes intuitive sense to anybody else. I always remember the alphabet, so alphabetical it's gotta be for me. Once I customize it for my need, it's reasonable to expect the app to remember my preference.

In general, many capabilities & defaults in MuseScore seem designed for the needs of musicians working in large formal ensembles like orchestras or concert bands. Nothing wrong with that, but I hope the team will extend MuseScore's capabilities to accommodate smaller, less formal configurations like improvisational jazz bands, jam sessions, singer-songwriter duos, etc. (I should make specific requests elsewhere, right?)

**Comment by MarcSabatella** [comment]
As you may have noticed, MU4 uses separate instances and hence separate windows for each score.  So by multiple windows, I mean multiple instances/scores.

And yes, if you have questions about or suggestions regarding how to best write music for smaller ensemble, the forum is the more appropriate place.  Then if a consensus developers around some sort of specific new enhancement that would make those cases easier, a new issue can be opened here.  Its preferred to keep the *discussion* (like this) on the forums, so the GitHub issue tracker can be more focused.

**Comment by avvvvve** [comment]
@jessjwilliamson Just FYI, it would be good to make sure this is resolved at the same time as the upcoming Workspaces designs.
